client/webserver/site/src/js/wallets.ts

```diff
@@ -603,7 +603,7 @@ export default class WalletsPage extends BasePage {
     const { bttn, tmpl } = this.assetButtons[assetID]
     Doc.hide(tmpl.fiat, tmpl.noWallet)
     bttn.classList.add('nowallet')
-    tmpl.img.src = Doc.logoPath(a.symbol)
+    tmpl.img.src ||= Doc.logoPath(a.symbol) // don't initiate GET if already set (e.g. update on some notification)
```

Comment:
I first did this with the nullish coalescing assignment operator (`&&=`) but to my surprise the `src` attribute is never null for an `img` tag, even if it's not there in the html; it's just empty, so we have to go with falsy.